desktop-environment.el

```diff
@@ -250,6 +303,19 @@ portion of the screen."
            (shell-command-to-string desktop-environment-volume-toggle-microphone-command)))
 
 
+;;; Commands - keyboard backlight
+;;;###autoload
+(defun desktop-environment-keyboard-backlight-increment ()
+  "Increment keyboard backlight by `desktop-environment-volume-normal-increment'."
```

Comment:
wrong variable name `desktop-environment-volume-normal-increment`.

Reply:
Changed function name.

Reply:
I don't know what you changed exactly, but I was referring to the docstring which still contains a reference to a volume-related variable instead of a keyboard-backlight one.